examples/basic-sc/README.md

```diff
@@ -50,107 +50,67 @@ objects would be set by the different personas in the system:
-
-```bash
-TEKTON_VERSION=0.30.0
-
-kapp deploy --yes -a tekton \
-	-f https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.30.0/release.yaml
-```
-
-- [kpack], for providing an opinionated way of continuously building container
+- [kpack](https://github.com/pivotal/kpack/blob/main/docs/install.md),
+  for providing an opinionated way of continuously building container
   images using buildpacks
 
-```bash
-KPACK_VERSION=0.4.3
-
-kapp deploy --yes -a kpack \
-	-f https://github.com/pivotal/kpack/releases/download/v$KPACK_VERSION/release-$KPACK_VERSION.yaml
-```
-
-- [source-controller], for providing the ability to find new commits to a git
+- [source-controller](https://fluxcd.io/docs/gitops-toolkit/source-watcher/#install-flux),
+  for providing the ability to find new commits to a git
   repository and making it internally available to other resources
 
-```bash
-kubectl create namespace gitops-toolkit
-
-# THIS CLUSTERROLEBINDING IS FOR DEMO PURPOSES ONLY - THIS WILL GRANT MORE PERMISSIONS THAN NECESSARY
-#
-kubectl create clusterrolebinding gitops-toolkit-admin \
-  --clusterrole=cluster-admin \
-  --serviceaccount=gitops-toolkit:default
-
-SOURCE_CONTROLLER_VERSION=0.17.0
-
-kapp deploy --yes -a gitops-toolkit \
-  --into-ns gitops-toolkit \
-  -f https://github.com/fluxcd/source-controller/releases/download/v$SOURCE_CONTROLLER_VERSION/source-controller.crds.yaml \
-  -f https://github.com/fluxcd/source-controller/releases/download/v$SOURCE_CONTROLLER_VERSION/source-controller.deployment.yaml
-```
-
-- [kapp-controller], for providing us with the ability of deploying multiple
+- [kapp-controller](https://carvel.dev/kapp-controller/docs/latest/install/),
+  for providing us with the ability of deploying multiple
   Kubernetes objects as a single unit
 
-```bash
-KAPP_CONTROLLER_VERSION=0.30.0
+- [knative-serving](https://knative.dev/docs/install/serving/install-serving-with-yaml/),
+  for being the runtime of the application we want to deploy.
 
-kapp deploy --yes -a kapp-controller \
-	-f https://github.com/vmware-tanzu/carvel-kapp-controller/releases/download/v$KAPP_CONTROLLER_VERSION/release.yml
-```
+4. A container image registry in which you have authorization to push images. The
+  credentials will be passed to kpack in the configuration steps below.
 
-- [knative-serving], for being the runtime of the application we want to deploy.
+5. [Tree](https://github.com/ahmetb/kubectl-tree), a tool that we will use to observe the objects created.
 
-```bash
-KNATIVE_SERVING_VERSION=0.26.0
+## Running the example in this directory
 
-kapp deploy --yes -a knative-serving \
-  -f https://github.com/knative/serving/releases/download/v$KNATIVE_SERVING_VERSION/serving-crds.yaml \
-  -f https://github.com/knative/serving/releases/download/v$KNATIVE_SERVING_VERSION/serving-core.yaml
-```
+This example uses two directories with sub-directories of Kubernetes resources:
+[../shared](../shared) and [this directory (.)](.).
```

Comment:
`this directory` reads confusing, could it be specific to `basic-sc` as the actual folder name?